src/schema/resolvers.ts

```diff
@@ -93,4 +95,10 @@ const validateLogin = async (email: string, password: string): Promise<OutUser>
   return user;
 };
 
+const authorizeAccess = (context) => {
+  if (context.userId === undefined) {
```

Comment:
```suggestion
  if (!context.userId) {
```

No Javascript entram mais valores nesse tipo de verificação (null, false ou qualquer outro valor que avalie pra false), porém ela é mais enxuta, e a ideia é que o userId seja um valor válido anyway né?

Reply:
Beleza! Tinha começado colocando assim, mas estava com um problema e acabei trocando em algum momento Provavelmente o problema era em outro lugar, porque agora funciona deixando assim.